ngraph/src/ngraph/runtime/reference/ctc_loss.hpp

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) 2020 Intel Corporation
```

Comment:
If we don't use this implementation in the nGraph library, please move it to test folder.

Reply:
FIxed